packages/react/src/components/header/components/headerLink/headerLinkDropdown/HeaderLinkDropdown.tsx

```diff
@@ -90,6 +90,14 @@ export const HeaderLinkDropdown = ({
 
   const childElements = getChildElementsEvenIfContainersInbetween(children);
 
+  const renderIcon = () => {
+    if (depth > 1 && dynamicPosition === DropdownMenuPosition.Right)
```

Comment:
This has not changed in this PR, but it is a bit weird to call this prop 'dynamicPosition', because it is not dynamic at all from the components point of view :) It's just a prop.

Reply:
Well, I guess.. 😄 I can rename it just to be `position`.